### ISSUE TITLE
Native Facebook query not returning counts

### ISSUE THREAD

**Issue by billerickson** [jaredatch/Shared-Counts#96]
This was first reported by Jon Brown. I've actually had similar reports from clients but assumed they were being rate-limited by Facebook and had them switch to SharedCount.com.

Our current GET request to Facebook only returns the object id (url), not the share counts. 

When I run [this code](https://gist.github.com/billerickson/32ce1829362c9a50eb7e6c250681a674) (pulled from Shared Counts plugin), here's the response from Facebook:

![screenshot](https://p198.p4.n0.cdn.getcloudapp.com/items/YEuAom9Z/Image+2020-03-01+at+9.40.52+AM.png?v=32aca5ee584cc2f0c353714da1f55101)

Adding `'fields' => 'og_object{engagement}'` to the `$args` ([full code here](https://gist.github.com/billerickson/50e4eab42cf851f6182b140b832fc895)) returns the following:

![screenshot](https://p198.p4.n0.cdn.getcloudapp.com/items/4gum9bBq/Image+2020-03-01+at+9.42.26+AM.png?v=de8abb23f29a6e7223cc968aa80fdad0)

Engagement appears to be the sum total of shares, comments, and reactions. I get the same number from SharedCount.com: 

![screenshot](https://p198.p4.n0.cdn.getcloudapp.com/items/8LuJPQ81/Image+2020-03-01+at+9.47.43+AM.png?v=ad955ac90823177d201392a5a683533b)

I'm not sure how to get the individual components though. Wondering if we should just update [total_count](https://github.com/jaredatch/Shared-Counts/blob/5e5a2f15a8102d07092100fd1fe444cf733da62f/includes/class-shared-counts-core.php#L662) if using the native source.

**Comment by jb510** [comment]
Thanks Bill. I couldn't for the life of me figure out FB's syntax. I presumed {} and og_object were placeholders, oops

It might also make sense at this time to update the call to include the API version number, to avoid unexpected breakage in the future, no?  I'll test that code.

https://developers.facebook.com/docs/graph-api/reference/v6.0/url

**Comment by NormanHoehne** [comment]
I've made a pull request #98 to face this issue

On https://developers.facebook.com/docs/graph-api/reference/v7.0/url there is an example response for the engagement on the supplied url. I tested different version numbers (beginning from 2.9) within the request url as @jb510 recommended but the requests and the responses did not differ from each other. That's why I did not included it in the pull request.
